ws-server/src/future.rs

```diff
@@ -45,11 +49,16 @@ use std::task::{Context, Poll};
 /// `select_with` providing some other future, the result of which you need.
 pub(crate) struct FutureDriver<F> {
 	futures: Vec<F>,
+	heartbeat: Interval,
```

Comment:
```suggestion
	stop_monitor_heartbeat: Interval,
```

maybe? Either way, some docs here would be good.